_posts/2015-02-19-Implicit_Mentorship.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: The importance of Implicit Mentorship
+category : Mentorship
+tagline: "What makes Implicit Mentorship different?"
+author: Steven McDonald
+tags: [Mentorship]
+---
+{% include JB/setup %}
+
+I think everyone understands that having an explicit mentor is a great thing that has many benefits but I sometimes I don't think that people think of Implicit Mentorship.
+
+Explicit Mentorship is an intentional process and is generally a relationship that is agreed on between two or more people. You can discuss topics with your mentor and get feedback and insights on various topics.
+
+##Implicit Mentorship?
+
+In the same way that a natural leader can emerge from certain situations I think mentors can also emerge. Often these people do not intend to mentor or even realize that they are mentoring someone else.
+
+Why is Implicit Mentorship important though? On a number of occassions I have seen situations where someone could greatly benefit from somebody else's experience. When making the suggestion that person X could learn from person Y, either person Y does not feel comfortable "teaching" or person X dismisses that they would gain anything of significance. However, if these people stumble onto the topic themselves, a conversation usually takes place and possibly even a transfer of knowledge no matter how small.
```

Comment:
I think you meant tough instead of though? (spelling)

I'm no expert on writing but my preference would be to start a new paragraph between significance & However. It seems this is moving away from why implicit mentorship is tough and dealing with what the benefits of it are.